modules/ui/sections/raw_membership_editor.js

```diff
@@ -376,6 +388,13 @@ export function uiSectionRawMembershipEditor(context) {
             .style('border-color', d => d.relation.tags.colour)
             .text(function(d) { return utilDisplayName(d.relation); });
 
+        labelEnter
+            .append('button')
+            .attr('class', 'members-download')
+            .attr('title', t('icons.download'))
+            .call(svgIcon('#iD-icon-load'))
+            .on('click', downloadMembers);
```

Comment:
Thanks for adding this button! This placement works, but I wonder if it would be less confusing to put it in the raw member editor, at the bottom next to the ➕ button. After all, that’ll be right where the user is seeing that some of the members haven’t downloaded yet. (Not a blocker for this PR.)